index.html

```diff
@@ -61,6 +61,172 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
+          <h3 class="underlined">My Recent Works</h3>
+        <div class="project-list">
+          <div class="project-item">
+            <div class="project-item-img">
+          <img src="/img/tonic2.png" alt="" class="img">
+        </div>
+      </div>
+      <div class="project-disc">
+      <div class="project-heading">Multi-Post Stories</div>
+      <p class="project-disc-main">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standar dummy text.</p>
+
+      <div class="project-btns">
+        <button class="project-btns-list">css</button>
+        <button class="project-btns-list">html</button>
+        <button class="project-btns-list" id="bts">bootstrap</button>
+        <button class="project-btns-list">Ruby</button>
+      </div>
+      <div class="project-bttn">
+        <button class="project-btn">See Project</button>
+      </div>
+      </div>
+          </div>
+        </div>
+        
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
```

Comment:
- [ ] As a good practice for professionals, as well as for improved web accessibility, including your button type is a good practice i.e. `<button type=""></button>` - which in this case is best if the `type` is set to `button`.